content/applications/sales/subscriptions/upselling.rst

```diff
@@ -1,86 +1,106 @@
-=====================
-Upsell a subscription
-=====================
+====================
+Upsell subscriptions
+====================
 
-Subscriptions are recurrent and go on indefinitely. As time passes by, our customers may want to
-modify them. We must then be able to adapt the prices or change the products’ quantities to
-accommodate their needs. Two situations can happen:
+Subscriptions are recurrent, and go on indefinitely. As time passes, customers may want to modify
+them. With that in mind, it's imperative to have the ability to adapt prices, or change quantities,
+to accommodate any need. That's where the opportunity to upsell a subscription can come into play.
 
-1. **Loyal customers:** This kind of customers already trust you as a brand. Therefore, you are
-   confident regarding what you offer since they keep paying for your products and services.
-   Consequently, it is easier to sell them something additional than it would be to a
-   new customer.
+Upselling may prove beneficial to the following customer types:
 
-2. **New customers:** For this kind of customers, you have to come with something new, something
-   attractive. What about discounts? Typically, every subscription ends after a certain given time.
-   Making these types of offers for new customers strengthens your relationships with them and also
-   increases their retention.
+#. *Loyal Customers*
+    These are customers who already trust the company/brand, and because they have established a
+    pattern of paying for products/services, there's more confidence behind the attempt to sell them
+    a more expensive product/service.
```

Comment:
@ksc-odoo if you want to [format a line break](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#line-blocks), use `|` at the start of the line and then indent a space for any lines that you want to wrap normally under the first line 🙂
```suggestion
#. | *Loyal Customers*
   | These are customers who already trust the company/brand, and because they have established a
     pattern of paying for products/services, there's more confidence behind the attempt to sell
     them a more expensive product/service.
```